### PR TITLE
[GDR-1975] data.frame-related functions

### PR DESCRIPTION
**Pull request by j-smola**
# Description
## What changed?
Related JIRA issue:  GDR-1975

## Why was it changed?
- change description of expected to data.table
- change format of data in rds file

# Checklist for sustainable code base
- [x] I added tests for any code changed/added
- [x] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [x] Package version bumped
- [x] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: tests/testthat/test-utils.R [hunk 1]
```diff
@@ -10,8 +10,6 @@ test_that("import_data", {
   template <- list.files(dataDir, pattern = "Template", full.names = TRUE)
   raw_data <- list.files(dataDir, pattern = "^RawData", full.names = TRUE)
   suppressWarnings(
-    expect_s3_class(import_data(manifest, template, raw_data), "data.frame")
+    expect_true(is(import_data(manifest, template, raw_data), "data.table"))
```
Comment (gladkia) [inline tests/testthat/test-utils.R:13]: FYI: https://github.com/gdrplatform/gDRutils/blob/master/tests/testthat/test-convert_mae_se_assay_to_dt.R#L157.

Comment (j-smola) [reply tests/testthat/test-utils.R:13]: 👍

File: R/import_data.R [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
                         results_file,
                         instrument = "EnVision") {
-  loaded_data <- load_data(manifest_file = manifest_file,
-                           df_template_files = template_file,
-                           results_file = results_file,
-                           instrument = instrument)
-  merge_data(manifest = loaded_data$manifest,
-             treatments = loaded_data$treatments,
-             data = loaded_data$data)
+  loaded_data <- gDRimport::load_data(manifest_file = manifest_file,
```
Comment (bczech) [inline R/import_data.R:22]: Namespacing is not required: https://github.com/gdrplatform/gDR/blob/ef48d2681079c21f96dd2a746573c06d0116d69e/R/package.R#L3-L4

File: R/import_data.R [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+                                      df_template_files = template_file,
+                                      results_file = results_file,
+                                      instrument = instrument)
+  gDRcore::merge_data(manifest = loaded_data$manifest,
```
Comment (bczech) [inline R/import_data.R:26]: as above: https://github.com/gdrplatform/gDR/blob/ef48d2681079c21f96dd2a746573c06d0116d69e/R/package.R#L3-L4

Comment (j-smola) [reply R/import_data.R:26]: ✅

File: tests/testthat/test-data.R [hunk 4]
```diff
@@ -0,0 +1,8 @@
+context("data")
```
Comment (gladkia) [inline tests/testthat/test-data.R:1]: 👍

**Review by gladkia — APPROVED** [review]

**Pull request merged**